openvdb/io/File.cc

```diff
@@ -71,10 +71,20 @@ struct File::Impl
     static Index64 getDefaultCopyMaxBytes()
     {
         Index64 result = DEFAULT_COPY_MAX_BYTES;
+#if defined(_MSC_VER)
+        char* s;
+        _dupenv_s(&s, nullptr, "OPENVDB_DISABLE_DELAYED_LOAD");
+        if (s) {
+            char* endptr = nullptr;
+            result = std::strtoul(s, &endptr, /*base=*/10);
```

Comment:
Maybe adjust the logic so it's no longer required to have this line twice? I'd also recommend putting the heap data straight into a unique_ptr to avoid any potential memory leaks here.